plugins/obs-ffmpeg/obs-ffmpeg.c

```diff
@@ -176,6 +183,12 @@ bool obs_module_load(void)
 		blog(LOG_INFO, "NVENC supported");
 		obs_register_encoder(&nvenc_encoder_info);
 	}
+#ifndef _WIN32
+	if (vaapi_supported()) {
+		blog(LOG_INFO, "FFMPEG VAAPI supported");
```

Comment:
"FFMPEG" should be "FFmpeg".

Reply:
Switched to matching existing NVENC style.